test/DocumentFormat.OpenXml.Tests/Spreadsheet/CellTests.cs

```diff
@@ -53,5 +53,30 @@ public void CellValidationTest(string value, CellValues type, bool isValid)
                 Assert.Single(results);
             }
         }
+
+        // Boolean
+        [InlineData(false, CellValues.Boolean, true)]
+        [InlineData(true, CellValues.Boolean, true)]
+        [Theory]
+        public void CellBooleanValueValidationTest(bool value, CellValues type, bool isValid)
+        {
+            var cell = new Cell
+            {
+                CellValue = new CellValue(value),
```

Comment:
These would make more sense in the `CellValueTests` class

Reply:
@twsouthwick in a sense yes, at the moment the CellBooleanTest is setting the string directly and correctly using "true" and "false" as InlineData, but the problem is that the constructor that gets the bool isn't being called, if you use that one it's going to write "True" and "False" which isn't valid.

I ment with "The test does fail if it runs with the old code" that my new tests fail because they are calling the new CellValue(true) and new CellValue(false)

I'm constructing the full Cell value because the CellValue itself doesn't really care how it's serialized, new CellValue(true) and new CellValue("true") would give the same result in the document. But when we combine it with the Cell having a CellValues.Boolean DataType the validator does check the cell value.

I've added a CellBooleanConstructorTest to test the constructor directly, in combination with the TryGetBoolean method.

Reply:
Put it differently, if we only take me 2 new test methods, which are correct, and not the change on the CellValue the tests will fail.

Reply:
That makes sense. Thanks for explaining!